src/components/BookItem.js

```diff
@@ -17,7 +17,10 @@ function BookItem({
         <button type="button">Comments</button>
         <button
           type="button"
-          onClick={() => dispatch(removeBook(itemId))}
+          onClick={() => {
+            dispatch(deleteBook(itemId));
+            dispatch(removeBook(itemId));
```

Comment:
**You did great job! Adding and removing is working as expected.** 

[OPTIONAL] It would be great idea if you make one `deleteBook` instead of `deleteBook` and `removeBook`. Because they do have one purpose, right :)

Reply:
@Rustamxon7 noted. I will explore implementing that. Thanks